### PR TITLE
Kubevirtci bump

### PR DESCRIPTION
**Pull request by brybacki**
**What this PR does / why we need it**:

Update kubevirtci so new kubernetes can be used in tests. 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by brybacki** [comment]
This PR only does a script cleanup and updates kubevirtci. Next PRs will try to update k8s to 1.21 or 1.22 and kubevirt to latest version. Tried to do it all at once, but then some tests start failing. I want to do it step by step to isolate problems.

File: cluster-up/cluster/k8s-1.16/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+Provides a pre-deployed Kubernetes with version 1.16.2 purely in docker
+containers with qemu. The provided VMs are completely ephemeral and are
+recreated on every cluster restart. The KubeVirt containers are built on the
+local machine and are then pushed to a registry which is exposed at
+`localhost:5000`.
+
+## Bringing the cluster up
+
+```bash
+export KUBEVIRT_PROVIDER=k8s-1.16.2
```
Comment (ShellyKa13) [inline cluster-up/cluster/k8s-1.16/README.md]: maybe change the example to newer version?

Comment (ShellyKa13) [reply cluster-up/cluster/k8s-1.16/README.md]: I see its specific for 1.16 ignore :)

File: cluster-up/hack/common.sh [hunk 2]
```diff
@@ -17,7 +17,7 @@ fi
 
 
 KUBEVIRTCI_CLUSTER_PATH=${KUBEVIRTCI_CLUSTER_PATH:-${KUBEVIRTCI_PATH}/cluster}
-KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER:-k8s-1.20}
+KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER:-k8s-1.19}
```
Comment (ShellyKa13) [inline cluster-up/hack/common.sh:20]: why? maybe explain in commit msg?

Comment (brybacki) [reply cluster-up/hack/common.sh:20]: good idea, added following messasge:
```
Updating everything (k8s, kubevirt, cdi, kubevirtci) resulted in failing tests. So it is better to do changes one by one one (to easier find what is wrong), so this change only updates kubevirtci and keeps kubevirt_provider as it was before at 1.19.
```

**Comment by brybacki** [comment]
good, tests now have 100% pass rate, no flakes

**Pull request merged**